modules/cluster-logging-vector-tech-preview.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-vector-tech-preview_{context}"]
+:FeatureName: Vector
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-vector"]
+= About Vector
+[role="_abstract"]
+Vector is an observability data pipeline currently offered as an alternative to the default collector for the {logging}.
+
+[IMPORTANT]
+====
+Vector does not support FIPS Enabled Clusters.
+====
+
+The following outputs are supported:
+
+--
+
+* `elasticsearch`. An external Elasticsearch instance. The `elasticsearch` output can use a TLS connection.
+
+* `kafka`. A Kafka broker. The `kafka` output can use an unsecured or TLS connection.
+
+* `loki`. Loki, a horizontally scalable, highly available, multi-tenant log aggregation system.
+
+
+[id="cluster-logging-enabling-vector"]
+= Enabling Vector
+[role="_abstract"]
+Vector is not enabled by default. Use the following steps to enable Vector on your {product-title} cluster.
+
+[IMPORTANT]
+====
+Vector does not support FIPS Enabled Clusters.
+====
+
+.Prerequisites
+
+* {product-title}: 4.10
+* {logging-title-uc}: 5.4
+* FIPS disabled
+
+.Procedure
+
+. Edit the `ClusterLogging` custom resource (CR) in the `openshift-logging` project:
++
+[source,terminal]
+----
+$ oc -n openshift-logging edit ClusterLogging instance
+----
+
+. Add annotation `logging.openshift.io/preview-vector-collector: enabled` to the `ClusterLogging` custom resource (CR).
+
+[source,yaml]
+----
+  apiVersion: "logging.openshift.io/v1"
+  kind: "ClusterLogging"
+  metadata:
+    name: "instance"
+    namespace: "openshift-logging"
+    annotations:
+      logging.openshift.io/preview-vector-collector: enabled
+  spec:
+    collection:
+  logStore:
+    type: "elasticsearch"
+    retentionPolicy:
+      application:
+        maxAge: 1d
+      infra:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
+    elasticsearch:
+      nodeCount: 3
+      resources:
+        limits:
+          memory: 16Gi
+        requests:
+          cpu: "1"
+          memory: 16Gi
+      storage:
+        storageClassName: "gp2"
+        size: "200G"
+      redundancyPolicy: "SingleRedundancy"
+  visualization:
+    type: "kibana"
+    kibana:
+      resources:
+        limits:
+          memory: 736Mi
+        requests:
+          cpu: 100m
+          memory: 736Mi
+      replicas: 1
+  collection:
+    logs:
+      type: "vector"
+      fluentd: {}
+        resources:
+          limits:
+            memory: 736Mi
+          requests:
+            cpu: 100m
+            memory: 736Mi
```

Comment:
please delete these lines, because we specied empty block `{}`

also, these lines have no effect even if these are configured. these will be removed in GA

Reply:
Truncated the code example to end after the annotation.